cmds/exp/smbios_transfer/smbios_transfer.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// The smbios_transfer command is used to send SMBIOS
```

Comment:
Write this comment in this format: https://github.com/u-root/u-root/blob/master/cmds/core/cat/cat.go#L7